src/main/java/org/homeschoolpebt/app/cli/TransmitterCommands.java

```diff
@@ -102,16 +114,24 @@ private void zipFiles(Map<String, Submission> appIdToSubmission, String zipFileN
         String subfolder = createSubfolderName(submission, transmission);
         try {
           if ("pebt".equals(submission.getFlow())) {
-            // generate applicant summary
-            byte[] file = pdfService.getFilledOutPDF(submission);
-
-            String fileName = pdfService.generatePdfName(submission);
-            zos.putNextEntry(new ZipEntry(subfolder));
-            ZipEntry entry = new ZipEntry(subfolder + fileName);
-            entry.setSize(file.length);
-            zos.putNextEntry(entry);
-            zos.write(file);
-            zos.closeEntry();
+            // delay 7 days if there aren't any uploaded docs associated with this application
+            Date submittedAt = submission.getSubmittedAt();
+            Date sevenDaysAgo = new Date(System.currentTimeMillis() - SEVEN_DAYS_IN_MS);
```

Comment:
Martha did some fancy date subtraction stuff in her recent work, you may wanna look at that if you're curious. (But this seems fine to me, and sometimes less fancy is better.)